src/design-system/settings/setting.component.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactNode } from 'react';
+
+import { Box } from '../box';
+import { Flex } from '../flex';
+import { Text } from '../text';
+
+import * as styles from './setting.css';
+
+interface SettingsItemProps {
+  addon?: ReactNode;
+  description: ReactNode;
+  onClick?: () => void;
+  testId?: string;
+  title?: ReactNode | string;
+}
+
+export const Setting = ({
+  description,
+  addon,
+  onClick,
+  testId = 'setting',
+  title,
+}: SettingsItemProps): React.ReactElement => (
+  <Box mb="$32" className={styles.root}>
+    <div className={styles.separator} />
+    <Box
+      pt="$32"
+      pb="$32"
+      pl="$16"
+      pr="$16"
```

Comment:
Q: can we at least allow classname or something to override this ? Maybe it won't be applicable in all the places. 
Or, remove a wrapper Box and control it from the parent if necessary?
Separator is an end-app choice.
Just thinking loudly... WDYT?